.github/workflows/test.yml

```
@@ -298,7 +298,7 @@ jobs:
 
   ovn-upgrade-e2e:
     name: Upgrade OVN from Base to PR branch based image
-    if: github.event_name != 'schedule'
+    if: ${{ false }}
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)